tests/director/tests/notifications/application_only_notifications_new_format_test.go

```diff
@@ -1419,3 +1422,138 @@ func TestFormationNotificationsWithApplicationOnlyParticipantsNewFormat(t *testi
+	t.Logf("Created application 2 with name %q and ID %q", appName2, app2ID)
+
+	t.Logf("Add webhook with type %q and mode: %q to application with type %q", graphql.WebhookTypeApplicationTenantMapping, graphql.WebhookModeAsyncCallback, appType)
+	op := operations.NewAddWebhookToObjectOperation(graphql.WebhookTypeApplicationTenantMapping, operations.WebhookReferenceObjectTypeApplication, app2ID, tnt).
+		WithWebhookMode(graphql.WebhookModeSync).
+		WithURLTemplate("{\\\"path\\\":\\\"" + conf.ExternalServicesMockMtlsSecuredURL + "/formation-callback/{{.TargetApplication.ID}}{{if eq .Operation \\\"unassign\\\"}}/{{.SourceApplication.ID}}{{end}}\\\",\\\"method\\\":\\\"{{if eq .Operation \\\"assign\\\"}}PATCH{{else}}DELETE{{end}}\\\"}").
+		WithInputTemplate("{\\\"context\\\":{\\\"crmId\\\":\\\"{{.CustomerTenantContext.CustomerID}}\\\",\\\"globalAccountId\\\":\\\"{{.CustomerTenantContext.AccountID}}\\\",\\\"uclFormationId\\\":\\\"{{.FormationID}}\\\",\\\"uclFormationName\\\":\\\"{{.Formation.Name}}\\\",\\\"operation\\\":\\\"{{.Operation}}\\\"},\\\"receiverTenant\\\":{\\\"state\\\":\\\"{{.Assignment.State}}\\\",\\\"uclAssignmentId\\\":\\\"{{.Assignment.ID}}\\\",\\\"applicationUrl\\\":\\\"{{.TargetApplication.BaseURL}}\\\",\\\"applicationTenantId\\\":\\\"{{.TargetApplication.LocalTenantID}}\\\",\\\"uclSystemName\\\":\\\"{{.TargetApplication.Name}}\\\",\\\"uclSystemTenantId\\\":\\\"{{.TargetApplication.ID}}\\\",\\\"configuration\\\":{{.Assignment.Value}}},\\\"assignedTenant\\\":{\\\"state\\\":\\\"{{.ReverseAssignment.State}}\\\",\\\"uclAssignmentId\\\":\\\"{{.ReverseAssignment.ID}}\\\",\\\"applicationUrl\\\":\\\"{{.SourceApplication.BaseURL}}\\\",\\\"applicationTenantId\\\":\\\"{{.SourceApplication.LocalTenantID}}\\\",\\\"uclSystemName\\\":\\\"{{.SourceApplication.Name}}\\\",\\\"uclSystemTenantId\\\":\\\"{{.SourceApplication.ID}}\\\",\\\"configuration\\\":{{.ReverseAssignment.Value}}}}").
+		WithOutputTemplate("{\\\"config\\\":\\\"{{.Body.config}}\\\", \\\"state\\\":\\\"{{.Body.state}}\\\", \\\"location\\\":\\\"{{.Headers.Location}}\\\",\\\"error\\\": \\\"{{.Body.error}}\\\",\\\"success_status_code\\\": 200}").Operation()
+	defer op.Cleanup(t, ctx, certSecuredGraphQLClient)
+	op.Execute(t, ctx, certSecuredGraphQLClient)
+
+	formationTmplName := "formation-template-name"
+	ftProvider := resource_providers.NewFormationTemplateCreator(formationTmplName)
+	defer ftProvider.Cleanup(t, ctx, certSecuredGraphQLClient)
+	ftplID := ftProvider.WithSupportedResources(appProvider1.GetResource(), appProvider2.GetResource()).
+		Provide(t, ctx, certSecuredGraphQLClient)
+	t.Logf("Created Formation Template with ID: %q and name: %q", ftplID, formationTmplName)
+
+	draftFormationName := "draft-formation-name"
+	t.Logf("Creating formation with name: %q from template with name: %q", draftFormationName, formationTmplName)
+	formationProvider := resource_providers.NewFormationProvider(draftFormationName, tnt, &formationTmplName).
+		WithState("DRAFT")
```

Comment:
Maybe move the "DRAFT" to a constant? We have already `readyAssignmentState` for example in the fixtures_test.go file.